configure.ac

```diff
@@ -2075,6 +2075,13 @@ else
     if test -z "$PYVER"; then
       PYVER=0
     fi
+
+    AC_MSG_CHECKING([for $PYTHON3 GIL is disabled])
+    PYGIL=`($PYTHON3 -c "import sysconfig; print(not bool(sysconfig.get_config_var('Py_GIL_DISABLED')))") 2>/dev/null`
+    AC_MSG_RESULT($PYGIL)
+    if test x"$PYGIL" = x"False"; then
+      PYTHON3="$PYTHON3 -Xgil=0"
+    fi
```

Comment:
This results in output:
```
checking for python3.13t GIL is disabled... False
```
Isn't the logic wrong here? It should surely output True. Then I don't think you need `-Xgil=0` to turn off the GIL if PYGIL is True. But if it is really needed the docs imply it should be with a space `-X gil=0` ... https://docs.python.org/3/using/cmdline.html#cmdoption-X.

We'll need to think of some way to show clearly that the GIL is turned off in the build or configure output, but probably the additional `AC_MSG_RESULT` is good enough once corrected.

Reply:
arrg...you are right, the comment needs to be other way around, fixed it. W.r.t. the command line option -Xgil=0, I think we need it because it ensures that the GIL stays disabled even if test cases import libraries, which are not explicitly build for the free threading case (the space between -X and gil is not needed), otherwise an import might enable the GIL back again.

Reply:
I made a few tweaks in 1b09f4f11112f0276f7b64c54e2a1ea73062fe52 to make it give more standard configure output.